### PR TITLE
Add operator-sdk annotations - FAR resources

### PR DESCRIPTION
**Pull request by razo7**
Include CRD's resources in CSV - customresourcedefinitions.
Follow up to [commit](https://github.com/medik8s/fence-agents-remediation/pull/16/commits/e959e90baa866336fec9d14f7a466b86e491973a) and [discussion](https://github.com/medik8s/fence-agents-remediation/pull/16#discussion_r1118518180).

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,6 +6,7 @@ require (
 	github.com/go-logr/logr v1.2.0
 	github.com/onsi/ginkgo v1.16.5
 	github.com/onsi/gomega v1.17.0
+	go.uber.org/zap v1.19.1
```
Comment (slintes) [inline go.mod:9]: this change doesn't make sense in the context of this PR?

Comment (slintes) [reply go.mod:9]: CI complains about the modified go.mod as well, so something is wrong, either the file, or CI. Check the make targets, order matters sometimes 🤷🏼‍♂️ (e.g. make generate and go fmt / goimports sometimes have slightly different output with import aliases)

Comment (mshitrit) [reply go.mod:9]: For me `go mod tidy` did the trick and reveled the changes

Comment (razo7) [reply go.mod:9]: I locally use Golang version 1.20. Maybe it matters

Comment (slintes) [reply go.mod:9]: It matters, always use the same (x.y at least) version locally.
I use https://github.com/travis-ci/gimme for quickly switching versions.

Comment (razo7) [reply go.mod]: Nice, I will give it a try. I will add the above change when I will update FAR to Golang v1.20 👍🏻

**Pull request merged**